app/src/views/RootLayout/index.tsx

```diff
@@ -56,6 +62,12 @@ export function Component() {
 
     const [languagePending, setLanguagePending] = useState(false);
 
+    const [isCookiesBannerVisible, setIsCookiesBannerVisible] = useState(true);
+
+    const handleClick = useCallback(() => {
+        setIsCookiesBannerVisible(false);
+    }, []);
+
```

Comment:
You can use useBooleanState (example at [app/src/views/CookiePolicy/index.tsx](https://github.com/IFRCGo/go-web-app/pull/1335/files#diff-3a7773dec67a20a95fc9518493f19aadb6ea41ef7c0741f1c97f678d9ca162d2))